src/sentry/rules/actions/notify_event_service.py

```diff
@@ -77,9 +83,9 @@ def after(self, event, state):
                 extra["group_id"] = group.id
                 self.logger.info("rule.fail.should_notify", extra=extra)
                 return
-
-            metrics.incr("notifications.sent", instance=plugin.slug, skip_internal=False)
-            yield self.future(plugin.rule_notify)
+            else:
```

Comment:
@MeredithAnya Is this `else` statement needed since we return in the other conditions? Maybe I am being a bit nitpicky, but it seems odd to me to have an `if`, `if`, then `else` like this while returning in the first two cases. I suggest removing the return statements and making this `if`, `elif`, and `else`, or just removing the `else`.